benchmarks/engines/busyring/arbor/ring.cpp

```diff
@@ -495,34 +498,34 @@ arb::cable_cell branch_cell(arb::cell_gid_type gid, const cell_parameters& param
         dist_from_soma += l;
     }
 
-    arb::label_dict d;
+    arb::label_dict labels;
 
     using arb::reg::tagged;
-    d.set("soma",      tagged(1));
-    d.set("dendrites", join(tagged(3), tagged(4)));
+    labels.set("soma",      tagged(1));
+    labels.set("dendrites", join(tagged(3), tagged(4)));
     if (params.synapses>1) {
-       d.set("synapses",  arb::ls::uniform(arb::reg::all(), 0, params.synapses-2, gid));
+       labels.set("synapses",  arb::ls::uniform(arb::reg::all(), 0, params.synapses-2, gid));
     }
 
-    arb::cable_cell cell(arb::morphology(tree), d);
+    arb::decor decor;
 
-    cell.paint("\"soma\"", "hh");
-    cell.paint("\"dendrites\"", "pas");
-    cell.default_parameters.axial_resistivity = 100; // [Ω·cm]
+    decor.paint("\"soma\"", "hh");
+    decor.paint("\"dendrites\"", "pas");
+    decor.set_default(arb::axial_resistivity{100}); // [Ω·cm]
 
     // Add spike threshold detector at the soma.
-    cell.place(arb::mlocation{0,0}, arb::threshold_detector{10});
+    decor.place(arb::mlocation{0,0}, arb::threshold_detector{10});
 
     // Add a synapse to the mid point of the first dendrite.
```

Comment:
Only issue is the comment here is out of date now (same as in Nora's PR).

Reply:
That's true. Never use comments, they lie.
I'll fix it.